src/kube-runtime/GOPATH/src/github.com/microsoft/runtime/Gopkg.toml

```diff
@@ -0,0 +1,11 @@
+[prune]
+  go-tests = true
+  unused-packages = true
+
+[[constraint]]
+  name = "gopkg.in/yaml.v2"
+  version = "2.2.2"
```

Comment:
why not use built-in yaml

Reply:
Don't find the build-in yaml lib. Do I miss something?

---
In reply to: [321699635](https://github.com/microsoft/pai/pull/3585#discussion_r321699635) [](ancestors = 321699635)